rector.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
I have used Rector to automatically upgrade code from PHP 7.4 to 8.0

Maybe I can simply remove this config as it was for single-use :thinking:

Reply:
je to si ce jednorázovka, ale použiješ to při dalším upgradu?